server/src/main/java/org/apache/druid/server/http/CoordinatorCompactionConfigsResource.java

```diff
@@ -168,6 +190,51 @@ public Response getCompactionConfig(@PathParam("dataSource") String dataSource)
     return Response.ok().entity(config).build();
   }
 
+  @GET
+  @Path("/{dataSource}/history")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getCompactionConfigHistory(
+      @PathParam("dataSource") String dataSource,
+      @QueryParam("interval") String interval,
+      @QueryParam("count") Integer count
+  )
+  {
+    Interval theInterval = interval == null ? null : Intervals.of(interval);
+    try {
+      List<AuditEntry> auditEntries;
+      if (theInterval == null && count != null) {
+        auditEntries = auditManager.fetchAuditHistory(
+            CoordinatorCompactionConfig.CONFIG_KEY,
+            CoordinatorCompactionConfig.CONFIG_KEY,
+            count
+        );
+      } else {
+        auditEntries = auditManager.fetchAuditHistory(
```

Comment:
Can a user pass in both interval and count?

Reply:
I copied the pattern in `CoordinatorDynamicConfigsResource#getDatasourceRuleHistory` where it looks like we ignore the count silently if the interval is specified.